modules/dod/module.json

```diff
@@ -2,7 +2,7 @@
 	"title": "Domains of Delight: A Feywild Accessory",
-	"description": "<p>Domains of Delight are to the Feywild what Domains of Dread are to the Shadowfell: sequestered realms governed by powerful beings. Whereas a Domain of Dread is ruled by a Darklord, a Domain of Delight is ruled by an archfey—the most powerful of Fey creatures. An archfey gives form to their Domain of Delight, shaping it in ways unique to their personality. Some Domains of Delight are bright and cheery, while others are gloomy, but each one reflects the emotional state of its ruler. A Domain of Delight can be as small as a few acres or as big as a country.</p><p>This accessory helps you create Domains of Delight and the archfey who rule them, building on the information about the Feywild that appears in the <em>Dungeon Master’s Guide</em>. The ideas, tips, and tables in this chapter are meant to spark your imagination. Use what excites and intrigues you, discard what doesn’t, and make up the rest!</p><ul><li><strong>Adds Detailed information and tips about the Feywild</strong> and how to use it in your game</li><li>Gives DMs <strong>everything they need to create their own Archfey and Domains of Delight</strong></li></ul>",
+	"description": "<p><strong>All proceeds from Domains of Delight go to non-profit Extra Life, an online grassroots movement working to save sick and injured kids through the power of play.</strong></p><p>Domains of Delight are to the Feywild what Domains of Dread are to the Shadowfell: sequestered realms governed by powerful beings. Whereas a Domain of Dread is ruled by a Darklord, a Domain of Delight is ruled by an archfey—the most powerful of Fey creatures. An archfey gives form to their Domain of Delight, shaping it in ways unique to their personality. Some Domains of Delight are bright and cheery, while others are gloomy, but each one reflects the emotional state of its ruler. A Domain of Delight can be as small as a few acres or as big as a country.</p><p>This accessory helps you create Domains of Delight and the archfey who rule them, building on the information about the Feywild that appears in the <em><a>Dungeon Master’s Guide</a></em>. The ideas, tips, and tables in this chapter are meant to spark your imagination. Use what excites and intrigues you, discard what doesn’t, and make up the rest!</p><ul><li><strong>Adds Detailed information and tips about the Feywild</strong> and how to use it in your game</li><li>Gives DMs <strong>everything they need to create their own Archfey and Domains of Delight</strong></li></ul><p>Discover the other books benefiting Extra Life:</p><ul><li><a>Infernal Machine Rebuild</a></li><li><a>One Grung Above</a></li><li><a>Locathah Rising</a></li><li><a>The Tortle Package</a></li><li><a>Lost Laboratory of Kwalish</a></li></ul><hr><p>Extra Life supports Children’s Miracle Network Hospitals® by raising funds and awareness that provide 32 million treatments each year to kids across the U.S. and Canada.</p>",
```

Comment:
Note that the module description does include the DMG title and list items which are missing in the README.